src/pgesv/HPL_pdgesvK2_no_stream.cpp

```diff
@@ -0,0 +1,245 @@
+/* 
```

Comment:
I suggest to have one version of the panel with a macro that enables/disables use of streams.